analyzers/tests/SonarAnalyzer.UnitTest/Rules/DoNotHideBaseClassMethodsTest.cs

```diff
@@ -25,14 +25,25 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class DoNotHideBaseClassMethodsTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<DoNotHideBaseClassMethods>();
+
         [TestMethod]
         public void DoNotHideBaseClassMethods() =>
-            new VerifierBuilder<DoNotHideBaseClassMethods>().AddPaths("DoNotHideBaseClassMethods.cs", "DoNotHideBaseClassMethods2.cs").WithAutogenerateConcurrentFiles(false).Verify();
+            builder.AddPaths(
+                "DoNotHideBaseClassMethods.cs",
+                "DoNotHideBaseClassMethods2.cs")
```

Comment:
I generally disagree with adding useless EOLS in all UTs.
Even this formatting wastes +2 for arguments that are not long.
```suggestion
            builder.AddPaths("DoNotHideBaseClassMethods.cs", "DoNotHideBaseClassMethods2.cs")
```